app/src/main/java/com/vultisig/wallet/ui/models/keysign/KeysignViewModel.kt

```diff
@@ -384,4 +402,19 @@ internal class KeysignViewModel(
             }
         }
     }
+
+    fun navigateToAddressBook() {
+        val sendTx = transactionTypeUiModel as? TransactionTypeUiModel.Send
+        sendTx?.tx?.let { tx ->
+            viewModelScope.launch {
+                navigator.route(
+                    Route.AddressEntry(
+                        vaultId = vault.id,
+                        address = tx.srcAddress,
```

Comment:
@aminsato , When adding an address to address book , we are adding "To" address to address book incase we need to send to the address frequently , it never is "srcAddress"